core/domain/wipeout_service_test.py

```diff
@@ -264,14 +457,321 @@ def test_delete_user_broken(self):
         ).put()
 
         self.assertFalse(wipeout_service.verify_user_deleted(
-            user_models.PendingDeletionRequestModel.get_by_id(self.user_2_id)))
+            wipeout_service.get_pending_deletion_request(self.user_2_id)))
+
+        wipeout_service.delete_user(
+            wipeout_service.get_pending_deletion_request(self.user_2_id))
+        self.assertTrue(wipeout_service.verify_user_deleted(
+            wipeout_service.get_pending_deletion_request(self.user_2_id)))
+
+
+class WipeoutServiceDeleteStoryModelsTests(test_utils.GenericTestBase):
+    """Provides testing of the deletion part of wipeout service."""
+
+    TOPIC_1_ID = 'topic_1_id'
+    STORY_1_ID = 'story_1_id'
+    STORY_2_ID = 'story_2_id'
+    USER_1_EMAIL = 'some@email.com'
+    USER_1_USERNAME = 'username1'
+    USER_2_EMAIL = 'some-other@email.com'
+    USER_2_USERNAME = 'username2'
+
+    def setUp(self):
+        super(WipeoutServiceDeleteStoryModelsTests, self).setUp()
+        self.signup(self.USER_1_EMAIL, self.USER_1_USERNAME)
+        self.signup(self.USER_2_EMAIL, self.USER_2_USERNAME)
+        self.user_1_id = self.get_user_id_from_email(self.USER_1_EMAIL)
+        self.user_2_id = self.get_user_id_from_email(self.USER_2_EMAIL)
+        self.save_new_topic(
+            self.TOPIC_1_ID,
+            self.user_1_id,
+            canonical_story_ids=[self.STORY_1_ID])
+        self.save_new_story(self.STORY_1_ID, self.user_1_id, self.TOPIC_1_ID)
+        wipeout_service.pre_delete_user(self.user_1_id)
+        wipeout_service.pre_delete_user(self.user_2_id)
+
+    def test_delete_story_simple(self):
```

Comment:
Try to make these test names a bit clearer and more descriptive -- see earlier comment.

Also try to write them more like English clauses, e.g. test_simple_story_is_deleted_when_...

Ditto below (and elsewhere).